src/pages/docs/support/process-dumps.md

```diff
@@ -55,6 +56,40 @@ Sometimes the deployment in Octopus can complete or error out too quickly, which
 ![Process explorer capturing child processes from Tentacle](/docs/support/images/processexplorer.png)
 :::
 
+## Creating process dumps on Linux
+To aid in debugging, process dumps on Linux should be captured using `createdump` or the `dotnet-dump` tool. Debuggers such as `gcore` and `gdb` produce dumps that are not portable, leading to difficulties diagnosing issues.
+
+`createdump` is included with installations of the dotnet runtime. You will first need to install the dotnet runtime by [following the instructions on the dotnet download page](https://dotnet.microsoft.com/en-us/download).
+
+Here is an example of a manual installation:
+```
```

Comment:
Where possible, add the language of the code sample as it will get syntax highlighting.

```suggestion
```bash
```